ExomeDepth/make_BEDdetail.py

```diff
@@ -49,6 +50,12 @@ def slice_vcf(args, merge_dic):
     event_dic = {}
     children = 0
     parents = 0
+
+    # Random shuffle to truly select random samples in subset (function calculate_statistics)
+    # Random shuffle has not influence on general statistics
```

Comment:
typo, 
additionally, is line 55 a code comment? maybe exclude this information

Reply:
comments removed.